pykafka/balancedconsumer.py

```diff
@@ -261,7 +271,7 @@ def _setup_zookeeper(self, zookeeper_connect, timeout):
         self._zookeeper = KazooClient(zookeeper_connect, timeout=timeout / 1000)
         self._zookeeper.start()
 
-    def _setup_internal_consumer(self, start=True):
+    def _setup_internal_consumer(self, partitions=[], start=True):
```

Comment:
Ouch, thanks for catching this - I'm a repeat offender with this one. Somehow my brain and python really don't agree on the semantics of kwargs :/